packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -172,6 +172,21 @@ describe('network stubbing', function () {
       this.testRoute(options, handler, expectedEvent, expectedRoute)
     })
 
+    // https://github.com/cypress-io/cypress/issues/8729
+    it('resolve ambiguity between overloaded definitions', () => {
+      cy.route2('POST', 'http://dummy.restapiexample.com/api/v1/create').as('create')
+
+      cy.window().then((win) => {
+        win.eval(
+          `fetch("http://dummy.restapiexample.com/api/v1/create", {
+            method: 'POST', // *GET, POST, PUT, DELETE, etc.
+          });`,
+        )
+      })
+
+      cy.wait('@create') // wait times out
```

Comment:
Remove comment about timing out
```suggestion
      cy.wait('@create')
```

Reply:
Deleted.